src/nvim/ui.c

```diff
@@ -155,7 +168,7 @@ bool ui_rgb_attached(void)
 
 bool ui_active(void)
 {
-  return ui_count != 0;
+  return ui_count > 1;
```

Comment:
is this because the internal/default grid is considered a UI? Or does this imply that the TUI is always enabled now?

Reply:
The compositor is an UI, and needs to be discounted. But this is really a clutch only because we both have dual `api/ui.c` and `ui_bridge.c` paths. Once the TUI uses msgpack and the bridge is burned, the compositor could be merged with `api/ui.c` module. (Note: this doesn't break the vimR interface, rather the contrary. The msgpack decoder should expose the same interface as the bridge does now to TUI+vimR, then vimR can vendor a compatible version of the decoder internally and stay compatible, even if nvim changes the decoder for TUI).